orchestrator/orchestrator/main.py

```diff
@@ -43,24 +61,128 @@ def get_instance_names() -> List[str]:
     return os.environ['NODES'].split()
 
 
-async def start_rnode_on(ssh_private_key_file_path: str, instance_hostname: Hostname) -> None:
-    command = 'docker pull rchain/rnode:latest'
-    async with asyncssh.connect(str(instance_hostname), client_keys=[ssh_private_key_file_path], known_hosts=None) as conn:
-        await conn.run(command, check=True)
+def render_command(fields: List[str]) -> str:
+    return ' '.join(shlex.quote(f) for f in fields)
+
+
+def make_rnode_command() -> List[str]:
+    shell_fields = [
+        'docker',
+        'run',
+        '--detach',
+        '--name=rnode',
+        '--network=host',
+        '--volume=/var/lib/rnode:/var/lib/rnode',
+        '--volume=/var/lib/rnode-static:/var/lib/rnode-static:ro',
+        '--volume=/usr/lib/x86_64-linux-gnu/libjemalloc.so.2:/opt/libjemalloc.so.2:ro',
+        '--entrypoint=/opt/docker/bin/rnode',
+        RNODE_DOCKER_IMAGE,
+        '--grpc-port=40401',
+        '-J-Xdebug',
+        '-J-Xrunjdwp:transport=dt_socket,address=127.0.0.1:8888,server=y,suspend=n',
+        '-XX:+HeapDumpOnOutOfMemoryError',
+        '-XX:HeapDumpPath=/root/heapdump_OOM.hprof',
+        '-XX:+ExitOnOutOfMemoryError',
+        '-XX:ErrorFile=/root/hs_err.log',
+        '-XX:MaxJavaStackTraceDepth=100000',
+        '-XX:NativeMemoryTracking=detail',
+        '-Dlogback.configurationFile=/var/lib/rnode-static/logback.xml',
+        '-p docker',
+        '-c /var/lib/rnode-static/rnode.conf',
+        'run',
+        '--network=hardnet1',
+        '--map-size=1099511627776',
+        '--kademlia-port=40404',
+        '--port=40400',
+        '--required-sigs=0',
+        '--store-type=v2',
```

Comment:
This is the default since a few moments so it can be omitted

Reply:
I'm using docker release tags here, so the default will take effect only after the next stable release.